src/book/03-guides/01-main-codebase/02-script-tuner.mdx

```diff
@@ -0,0 +1,99 @@
+
+The first line tells you the name and path of the script you are creating or editing.
+
+The second line specifies the number of frames and the current frame, which is highlighted. Each frame specifies different joint positions, and when these frames are combined they result in a motion.
+
+The next line is the duration. This determines how long the robot will take to move to the servo positions you have specified. This is measured in milliseconds.
+
+The next section is a table. These show the angle and gain for each servo for that frame.
+
+The bottom of the interface shows what commands are available. It does not show all the commands. All commands can be viewed by typing `:help`. Note that the letters are UPPERCASE.
+
+| Command                        | Description                                                                                                                                                                                                                                                                                                                                                                                          |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <kbd>,</kbd>                   | Moves to the frame that is left of the frame you have currently selected. When you move to the frame, the robot's joints will move to the joint positions for that frame.                                                                                                                                                                                                                            |
+| <kbd>.</kbd>                   | Moves to the frame that is right of the frame you have currently selected. When you move to the frame, the robot's joints will move to the joint positions for that frame.                                                                                                                                                                                                                           |
+| <kbd>N</kbd>                   | Creates a new frame after the frame you have currently selected. This frame will be a duplicate of the frame you have selected. It will be added in after the currently selected frame, and the selected frame will not change.                                                                                                                                                                      |
```

Comment:
```suggestion
| <kbd>N</kbd>                   | Creates a new frame after the currently selected frame. This frame will be a duplicate of the selected frame. It will be added in after the currently selected frame, and the selected frame will not change.                                                                                                                                                                      |
```